python-main.js

```diff
@@ -1178,6 +1182,19 @@ function web_editor(config) {
             // Stop immediate closure
             e.stopImmediatePropagation();
         });
+
+        window.addEventListener('resize', function(){
+            if($(".helpsupport_container").is(":visible") && $(".helpsupport_container").offset().left !== $("#command-help").offset().left
+            && $(".helpsupport_container").offset().left && $(window).width() > 620){
+                $(".helpsupport_container").css("top", $("#command-help").offset().top + $("#toolbox").height() + 10);
+                $(".helpsupport_container").css("left", $("#command-help").offset().left);
+        }
+            else if($(window).width() < 620 && $(".helpsupport_container").is(":visible")){
+                $(".helpsupport_container").css("left", $("#command-help").offset().left- $(".helpsupport_container").width() + 60);
+                $(".helpsupport_container").css("top", $("#command-help").offset().top + $("#toolbox").height() + 10);
+            }
+        });
+
```

Comment:
Since these two blocks of code a very similar, one last thing you can do is extract them into a function (you can define it inside `setupButtons` before `$("#command-help").click()`), and then have both `$("#command-help").click()` and `window.addEventListener('resize',...)` use the same function.

Reply:
Done